.coveragerc

```diff
@@ -3,6 +3,7 @@ source = seirmo
 omit = 
     seirmo/tests/*
     seirmo/version_info.py
+    seirmo/slider.py
```

Comment:
As discussed, do you think we might be able to test the sliders class with 100% coverage?